core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -2127,9 +2124,13 @@ class WorkspaceService(protected val userInfo: UserInfo, val dataSource: SlickDa
                   _ <- createWorkflowCollectionFuture
                   _ <- syncPoliciesFuture
                 } yield()})
+              (googleProjectId, googleProjectNumber) <- traceDBIOWithParent("setupGoogleProject", parentSpan)(_ => DBIO.from(
+                setupGoogleProject(billingProject, billingAccount, workspaceId, workspaceName, policyEmailsByName, billingProjectOwnerPolicyEmail, parentSpan)))
+              savedWorkspace <- traceDBIOWithParent("saveNewWorkspace", parentSpan)(span =>
+                createWorkspaceInDatabase(workspaceId, workspaceRequest, bucketName, billingProjectOwnerPolicyEmail, googleProjectId, Option(googleProjectNumber), Option(billingAccount), dataAccess, span))
               // After the workspace has been created, create the google-project resource in Sam with the workspace as the resource parent
               _ <- traceDBIOWithParent("createResourceFull (google project)", parentSpan)(_ => DBIO.from(
-                  samDAO.createResourceFull(SamResourceTypeNames.googleProject, googleProjectId.value, Map.empty, workspaceRequest.authorizationDomain.getOrElse(Set.empty).map(_.membersGroupName.value), userInfo, Option(SamFullyQualifiedResourceId(workspaceId, SamResourceTypeNames.workspace.value)))))
+                  samDAO.createResourceFull(SamResourceTypeNames.googleProject, googleProjectId.value, Map.empty, Set.empty, userInfo, Option(SamFullyQualifiedResourceId(workspaceId, SamResourceTypeNames.workspace.value)))))
```

Comment:
google project should not have auth domain, since permissions are handled by permissions on the workspace, via hierarchical resources

fixes this test failure:

```
[info]   org.broadinstitute.dsde.workbench.service.RestException: {
[info]   "causes": [],
[info]   "message": "http error calling uri https://sam-fiab.dsde-dev.broadinstitute.org/api/resources/v1/google-project",
[info]   "source": "rawls",
[info]   "stackTrace": [],
[info]   "statusCode": 400,

which looks like this in the sam logs:

[INFO] [05/13/2021 14:40:19.830] [scala-execution-context-global-2083] [akka.actor.ActorSystemImpl(sam)] HttpMethod(POST) http://app:8080/api/resources/v1/google-project: 400 Bad Request entity: {"causes":[{"causes":[{"causes":[],"message":"Auth Domain is not permitted on resource of type: google-project","source":"sam","stackTrace":[]}],"message":"Invalid Auth Domain specified","source":"sam","stackTrace":[]}],"message":"Cannot create resource","source":"sam","stackTrace":[],"statusCode":400}
```